versions/wmde1.env

```diff
@@ -5,7 +5,7 @@ MEDIAWIKI_BRANCH_NAME=REL1_35
 WDQS_VERSION=0.3.40
 WDQS_FRONTEND_COMMIT_HASH=422b00acbacdfcdf343b5f6532f82f89f4f4e1c2
 
-QUICKSTATEMENTS_COMMIT_HASH=f3d5ff092284949a9880747f1259b343f62a2ce9
+QUICKSTATEMENTS_COMMIT_HASH=9079e2c0179f1377853e77b1ba3daec5839ebb45
```

Comment:
Was this changed on purpose?

Reply:
I believe it simply includes the updates quickstatements version: https://github.com/magnusmanske/quickstatements/commit/9079e2c0179f1377853e77b1ba3daec5839ebb45

probably it would have been more clean to apply it separately, but this quickstatement chase has really been a dynamic adventture